web/themes/custom/usecaselabs/templates/block/block--block-content--type--modal-dialog.html.twig

```diff
@@ -0,0 +1,62 @@
+{#
+/**
+ * @file
+ * Olivero's implementation to display a block.
```

Comment:
@imudasirali Replace with the custom theme name

Reply:
Changed @sabreena